weekly-updates/Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -21,6 +22,174 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
+* Project scoping meeting w DS4S&H
+  * Open canvas for the win!
+* Feedback on Turing's Slack workspace.
+* TPS Seminar w DT 🌟
+* Brainhack paper published!
+  * 📑 https://doi.org/10.1016/j.neuron.2021.04.001
+  * 🧵 https://twitter.com/brainhackorg/status/1388147807958315011?s=20
+
+#### Also rans 😅
+
+* AutSPACEs Fortnightly Meet Ups x2
+  * Also did some strategic work on aligning these meet ups to our project needs.
+* Supported lab member to take time that they needed.
+* Mentorship call w SF to brainstorm future opportunities.
+* Got content written for a briefing note for a potential collaboration.
+  * Although very very late so ... not my best 😩
+* DECOVID meetings (knowledge shares, all hands, steering group, leadership, comms)
+* Supported data extracts.
+* Feedback on website content update.
+* I did some good escalating of an ongoing risk and made _some_ progress towards mitigating it.
+  * One person has very heavily come to my assistance on the plan 🙏
+* Fed into government review for the use of health data.
+* BIDS letter of intent for CZI application.
+* BIDS Steering Group meeting.
+  * Only one though, I missed another because my calendar wasn't up to date.
+* Made _some_ progress on job descriptions... but not as much as I'd hoped.
+* Programme directors meeting.
+* RIAC meeting.
+* ASG Theme Leads meeting.
+  * & follow up w BMA.
+* Coached someone for an interview & said fun and motivational things 💪
+  * Coached someone else for a promotion and said very similar fun and motivational things to them too 🙌
+* Had a fun call with someone who's struggling and gave some advice on how to collaborate without feeling like it's super overwhelming.
+* Catch up w ML.
+* Meetings re: ongoing partnership.
+* Meeting with Jupyter committee re: event funding.
+  * Follow up meeting w TA
+* Gave feedback on funding application for Raphtory team.
+* Attended D&S meet up
+* Call w Ada Lovelace Institute re: letter & suggestions.
+* Helpful mentorship chat w TG
+* Fabulous mentorship chat w MM
+* Supported submission of a TPS sponsored grant 🤞😬🤞
+* Reviewed & sent off short bio for DS4S&H grant as co-i 🤞😬🤞
+* 2 x Turing fellow project meeting re: TPS opportunities.
+* TPS & PPP project scoping meeting.
+* 3 x meetings for potential partnerships
+  * Good prep and follow up for one that will likely be led by TPS.
+  * Well done to AM for prepping me for that really well.
+* EDoN Analytics Hub meeting.
+* Turing Way meeting w Cambridge reproducibility leads
+* Said yes to being on a programme committee... that feels... like a bad idea.
+* Advocated for money for business team member to attend a conference.
+* Code of conduct violation, took report and had conversation.
+* Reviewed TPS and Turing Way newsletters! 💌💌
+* 1:1s w MS, IKG, AB, AM, GA, LC, ME, EK,
+* Lab meetings
+  * Overview of CollabW21 🤝
+  * Ethics form 🦖
+  * Goatherding 🐐
+  * Community management 🌻
+
+### What did you struggle with?
+
+* Onboarding documentation is very poor.
```

Comment:
Lab onboarding documentation or other onboarding documentation?

Reply:
Lab onboarding. Just got to find the time to pull it together _sigh_.